hack/pre-release.sh

```diff
@@ -0,0 +1,72 @@
+# this script updates the WMCO and OCP versions in the Makefile, base CSV, bundle and Dockerfiles.
+
+# usage: 
+# ./pre-release (x, y or z) 
+# to update a z stream, use ./pre-release.sh z
+# for other streams, use the corresponding x y or z argument.
```

Comment:
It is unclear how to use this. Do you literally mean `x`, `y` or `z`? Is there an assumption that you will be in a `release-4.x` branch when executing this?